csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -565,8 +565,8 @@ The properties are: `names`, `organization`, `summary`, and `urls`.
 
 #### 3.1.1.1 Acknowledgments Type - Names
 
-List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of entities being recognized.
-Every such item of value type `string` with 1 or more characters represents the name of entity being recognized and contains the name of a single person.
+List of acknowledged names (`names`) has value type `array` with 1 or more items holds the names of persons being recognized.
```

Comment:
Or, `contributors`? … the plural of person reads not so good to me, esp. when lacking a relating `the` - maybe my acquired language module just fails me here.